src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/importer/LocationType.kt

```diff
@@ -9,24 +9,36 @@ import uk.gov.justice.digital.hmpps.pecs.jpc.location.Location
 private const val TYPE = 1
 private const val SITE = 2
 private const val AGENCY = 3
+private const val ACTIVE_INACTIVE = 6
 
-enum class LocationType(val index: Int, val label: String) {
+enum class LocationType(val label: String, private val activeFieldPosition: Int = ACTIVE_INACTIVE) {
 
-    COURT(5, "COURT"),
-    HOSPITAL(9, "HOSPITALS"),
-    IMMIGRATION(10, "IMMIGRATION"),
-    OTHER(12, "OTHER"),
-    POLICE(6, "POLICE"),
-    PRISON(8, "PRISON"),
-    STCSCH(11, "STC&SCH");
+    COURT("Courts", 8), // The position of this cell in the court tab column differs to the majority by two.
+    HOSPITAL("Hospitals"),
+    IMMIGRATION("Immigration"),
+    OTHER("Other"),
+    POLICE("Police", 7), // The position of this cell in the police tab column differs to the majority by one.
+    PRISON("Prisons"),
+    STCSCH("STC&SCH");
 
-    fun toLocation(cells: List<Cell>) : Location =
-            Location(
-                    locationType = cells[TYPE].stringCellValue.toUpperCase().trim(),
-                    nomisAgencyId = cells[AGENCY].stringCellValue.trim(),
-                    siteName = cells[SITE].stringCellValue.toUpperCase().trim())
+    /**
+     * Will return null if location is deemed to be not active.
+     */
+    fun active(cells: List<Cell>): Location? {
+        val active = "ACTIVE" == cells[this.activeFieldPosition].stringCellValue.toUpperCase().trim()
 
-    fun toLocation(cells: Row) : Location = toLocation(cells.toList())
+        return active.then(Location(
```

Comment:
looks like you could write `active then(Location ...)` since this is a infix function (not sure that improves readability)